WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -647,6 +666,100 @@ class AbstractPostListViewController : UIViewController, WPContentSyncHelperDele
         presentViewController(navController, animated: true, completion: nil)
     }
 
+    // MARK: - Searching
+
+    func toggleSearch() {
+        searchController.active = !searchController.active
+    }
+
+    func heightForSearchWrapperView() -> Float {
```

Comment:
I realize it was this way before, but everywhere `heightForSearchWrapperView` is called the result is being cast to a `CGFloat`.  Maybe fix the type and removing the casting if its convenient with the refactor?